validator_client/src/block_service.rs

```diff
@@ -852,6 +845,44 @@ impl<T: SlotClock + 'static, E: EthSpec> BlockService<T, E> {
 
         Ok::<_, BlockError>(unsigned_block)
     }
+
+    /// Translate `builder_proposals``, `builder_boost_factor`` and `prefer_builder_proposals`` to a
```

Comment:
```suggestion
    /// Translate `builder_proposals`, `builder_boost_factor` and `prefer_builder_proposals`` to a
```

Reply:
Updated in https://github.com/sigp/lighthouse/pull/5086/commits/b225328f9dd6a0bcfcfe83229b58d8774f9771d5